lib/modules/datasource/maven/index.spec.ts

```diff
@@ -1,15 +1,19 @@
+// eslint-disable-next-line import/order
```

Comment:
why?

Reply:
Because if it's not included at the top, Jest can't mock the S3 library. If it's moved further down, the tests fail.  Happy to change it if there's an obvious solution to it though.